modules/kubernetes/pages/deployment-concerns.adoc

```diff
@@ -0,0 +1,87 @@
+= Addressing deployment challenges on Kubernetes
+
+When you create a site on Kubernetes, there are many options you can use, for example, the number of pods for each component and the resource allocations for the associated pods.
+This guide focusses on the following goals:
+
+* xref:increased-traffic[]
+* xref:high-availability[]
+
+// Type:procedure
+[id="increased-traffic"]
+== Scaling for increased traffic
+
+For optimal network latency and throughput, you can adjust the CPU allocation for the router.
+Adjusting other allocations does not affect network performance.
+
+NOTE: Increasing the number of routers does not improve network performance.
+
+. Determine the router CPU allocation you require.
++
+--
+By default the router CPU allocation is `BestEffort`. 
+
+At a minimum, set the allocation to `1` to avoid issues with performance.
+
+For production environments, set the allocation to `2`.
+
+For maximum performance, set the allocation to `5`.
+
+--
+
+. If you are using the `skupper` CLI, set the CPU allocation for the router using the `--router-cpu` option, for example:
++
+--
+[source, bash]
+----
+$ skupper init --router-cpu 2
+----
+--
+
+. If you are using YAML, set the CPU allocation for the router by setting a value for `router-cpu`, for example:
++
+[source, YAML]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "skupper-site"
+data:
+  name: "my-site"
+  router-cpu: 2
+----
+--
+
+
+// Type:procedure
+[id="high-availability"]
+== Creating a high availability site
+
+By default, Kubernetes restarts any router that become unresponsive.
```

Comment:
By default, Kubernetes restarts any router that become**s**

Reply:
Should this not refer to downstream ie. OCP?

Reply:
no, OCP is only one option